inst/bs3compat/_headers.scss

```diff
@@ -2,9 +2,15 @@
 // which is a pretty jarring change, expecially for Rmd
 // https://github.com/twbs/bootstrap/blob/f17f882d/dist/css/bootstrap.css#L1219-L1250
 // https://github.com/twbs/bootstrap/blob/4de4874e/scss/_reboot.scss#L82-L89
-h1, h2, h3 {
-  margin-top: 1.5 * $spacer;
-}
-h4, h5, h6 {
-  margin-top: $spacer;
+
+@if $bootstrap-version >= 5 {
+  .section.level1, .section.level2, .section.level3 {
+    margin-top: 1.5 * $spacer;
+  }
+  .section.level4, .section.level5, .section.level6 {
+    margin-top: $spacer;
+  }
```

Comment:
A few things to double-check:

 - This works with recent versions of pandoc
 - What are the ramifications for the fact that you can turn off section divs?
 - Does this (or something similar) work for both html4 and html5?

Reply:
If we want this to work for pandoc-generated HTML in both html4 and html5 outputs, we'll need to include complimentary rules. A rule like `.section.level1` should have a `section.level1` counterpart to be applied in html5.

Reply:
> What are the ramifications for the fact that you can turn off `--section-divs`?

In Bootstrap 3 and 4, because the `margin-top` is applied to the `<hX>` tags, there's no difference between `section_divs = TRUE` or `section_divs = FALSE` or html4 or html5 outputs.

With Bootstrap 5, however, turning off `section_divs` will remove the margin-top above `<hX>` elements because our pandoc-specific styles won't apply.